src/System.Net.Security/src/System/Net/Security/SslStreamInternal.cs

```diff
@@ -56,12 +56,22 @@ private void ReturnReadBufferIfEmpty()
         }
 
         ~SslStreamInternal()
+        {
+            Close(disposing: false);
+        }
+
+        internal void Close(bool disposing)
         {
```

Comment:
Nit:
I think it'd be easier to understand if this were closer to the standard Dispose pattern:
- have two methods: public Dispose() and private Dispose(bool)
- the finalizer calls Dispose(false)
- Dispose calls Dispose(true) and SuppressFinalize
- SslState calls Dispose()

Reply:
The Dispose(bool) overload is not going to use the bool value at all, it's just going to be clearing the internalbuffer in both cases.. Can change it if that pattern must be absolutely followed..